index.bs

```diff
@@ -1012,6 +1032,33 @@ The {{ScopedCredentialEntity}} dictionary describes a user account or a [=relyin
         a user's avatar or a [=relying party=]'s logo.
 </div>
 
+
+### Authenticator Selection Criteria ### {#authenticatorSelection}
+
+The {{AuthenticatorSelectionCriteria}} dictionary describes a set of criterias for which the [RPS] can use to find the appropriate
+authenticator 
+
+<xmp class="idl">
+    dictionary AuthenticatorSelectionCriteria {
+        Attachment    attachment;
+        boolean       requireResidentKey;
```

Comment:
In this terminology, and in the ["Client-side-resident Credential Private Key"](https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/AngeloKai/webauthn/angelo-secondAuthnrSelect/index.bs#client-side-resident-credential-private-key) term above, I'd still rather focus on the fact that this allows creating credentials using just the RP Id without passing a Credential Id. The fact that the key will be stored client-side is just an implementation detail of that RP-visible feature.